hack/convert_manifest_to_jsonnet.py

```diff
@@ -0,0 +1,29 @@
+"""A simple utility to convert a manifest to corresponding jsonnet."""
```

Comment:
Parts of this script seem brittle, what's the purpose of it?

Reply:
It was a hack hence its location in "hacks" for converting our existing yaml manifests to jsonnet.
In a subsequent PR when we clean up the old stuff and move to ksonnet we can delete it.